src/pages/PageHome.jsx

```
@@ -18,20 +21,31 @@ export default function PageHome() {
+        <select onChange={handleMovieListsChange}>
+          <option value="filter_options">Filter Options</option>
+          <option value="now_playing">Now Playing</option>
+          <option value="popular">Popular</option>
+          <option value="top_rated">Top Rated</option>
+          <option value="upcoming">Upcoming</option>
+        </select>
+        <ul>
+          {data?.results &&
+            data.results.map((movieDetails) => (
+              <MovieCard key={movieDetails.id} details={movieDetails} />
+            ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)